src/main/java/seedu/address/model/person/Person.java

```diff
@@ -112,6 +121,10 @@ public String toString() {
                 .append("; Address: ")
                 .append(getAddress());
 
+        if (getProperty().isPresent()) {
+            builder.append(getProperty().get());
+        }
```

Comment:
Here is the new example output of `toString()` for a `Person`:
`Elon Gate; Phone: 12345; Email: elon@lelon.elon; Address: Woodlands; Property: [1 Woodlands Drive; Region: North; Size: 5-room; Price: $5.6M]; Tags: [Seller]`

This causes the Ui to display the property information with surrounding square brackets, but i think that is fine for now
![image](https://user-images.githubusercontent.com/45669732/157266363-a81e39bd-6588-4f58-90dc-359f91a1d47d.png)

Reply:
Looks good! Yeah I think the presentation of the property details shouldn't be too big of a deal now, we can flag it as a low priority issue since it's mostly cosmetic. We can discuss this with the group & get everyone's opinion 

It's something we can revisit again in the future 😄